spring-scaffold-cli/pom.xml

```diff
@@ -26,7 +26,7 @@
 		<dependency>
 			<groupId>br.com</groupId>
 			<artifactId>spring-boot-generate</artifactId>
-			<version>0.0.1-SNAPSHOT</version>
+			<version>0.0.2-SNAPSHOT</version>
```

Comment:
Consider using a more specific versioning scheme, such as semantic versioning, instead of `0.0.2-SNAPSHOT` to better track changes and releases.